<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishPoundageWidget</class>
 <widget class="QWidget" name="PublishPoundageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>360</y>
     <width>96</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>96</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>135</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发行账号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_accounts">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>195</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LNK 数量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>135</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>承兑数量:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>195</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>承兑币种:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_targetType">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>255</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>手续费:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fee">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20 LNK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_source">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>225</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_target">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>164</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>144</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>674</width>
     <height>287</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_back</zorder>
  <zorder>okBtn</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>label</zorder>
  <zorder>comboBox_accounts</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>comboBox_targetType</zorder>
  <zorder>label_5</zorder>
  <zorder>label_fee</zorder>
  <zorder>lineEdit_source</zorder>
  <zorder>lineEdit_target</zorder>
  <zorder>line_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
